new about us page

MyDanes opened the pull request:
new about us page added

MyDanes committed:
new about us page

new about us page added

cwinters8 commented on about/index.html:
```diff
@@ -0,0 +1,52 @@
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>About Doodle Dandy  | Doodle Dandy</title>
```
You have too many spaces here - there should be one space before and after the pipe character (`|`). Not sure if you want to say `About Doodle Dandy` or just `About us` since Doodle Dandy is already in the title?

cwinters8 commented on about/index.html:
```diff
@@ -0,0 +1,52 @@
+      <h1>About us</h1>
+      <p>
+        Our mission is to raise healthy, happy, and well-socialized pups so you can experience the joy of owning a Doodle. Every puppy is raised in our home—none of our dogs, including moms, pups, and papa dogs, live in a kennel. These dogs have brought immense happiness into our lives, and it's a privilege to share that love and companionship with others through responsible breeding. Let us help ...
+
+
```
These blank lines need to go.

cwinters8 commented on about/index.html:
```diff
@@ -0,0 +1,52 @@
+    <main>
+      <h1>About us</h1>
+      <p>
+        Our mission is to raise healthy, happy, and well-socialized pups so you can experience the joy of owning a Doodle. Every puppy is raised in our home—none of our dogs, including moms, pups, and papa dogs, live in a kennel. These dogs have brought immense happiness into our lives, and it's a privilege to share that love and companionship with others through responsible breeding. Let us help ...
```
You're only doing `AussieDoodles`? What does `Extend-A-Doodle` mean?

You should have a space before and after the dashes in the two sentences that contain them.

cwinters8 commented on about/index.html:
```diff
@@ -0,0 +1,52 @@
+
+
+      </p>
+      <ul>
+        <li>
+                 
+        </li>
+      </ul><img src="" alt="">
```
What is this supposed to be for?

cwinters8 committed:
fix formatting

MyDanes committed:
New Breed page

Adding new breed page 9.4.2024

cwinters8 commented on breed/index.html:
```diff
@@ -0,0 +1,101 @@
+        Poodle with an Australian Shepherd. This charming mix combines the best
+        traits of both parent breeds, resulting in a dog that is not only
+        incredibly adorable but also highly intelligent and playful.
+        <img
+          src="../assets/images/ddpup2.jpg"
+          alt="Doodle Dandy Puppy Love"
+          class="dog"
+        />
```
Images should not be inside paragraph elements. You should do separate p elements for each of the text sections and keep the images outside of them. You'll get rid of the br elements as well.

cwinters8 commented on index.html:
```diff
@@ -32,13 +32,15 @@
       <nav>
         <a href="/" class="current">Home</a>
         <a href="/contact">Contact us</a>
-      </nav>
```
This needs to be put back at the end of your links.

MyDanes committed:
Fix broken ps

Fix Broken paragraphs remove Brs

MyDanes committed:
added current litter 9.4.2024

also updated links on home

cwinters8 commented on breed/index.html:
```diff
@@ -0,0 +1,106 @@
+      </nav>
+    </header>
+    <main>
+      <h1>AussieDoodle Breed Info</h1>
+
+      <p>
+        The Aussiedoodle is a delightful hybrid breed created by crossing a
```
Decide if its `AussieDoodle`, `Aussiedoodle`, or `Aussie Doodle`. I suspect the last one is actually the correct form.

cwinters8 replied:
There are other places where the name is used, so update it across the board when you decide.

MyDanes replied:
Both done and changes commited

cwinters8 committed:
improve home page styling and links

cwinters8 committed:
add some space at the bottom of the body

cwinters8 committed:
center dog images

cwinters8 commented on breed/index.html:
```diff
@@ -0,0 +1,106 @@
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>About Doodle Dandy | Doodle Dandy</title>
```
This should either be `About | Doodle Dandy` or `About us | Doodle Dandy`

cwinters8 commented on about/index.html:
```diff
@@ -0,0 +1,60 @@
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>About Doodle Dandy | Doodle Dandy</title>
```
This should either be `About | Doodle Dandy` or `About us | Doodle Dandy`

MyDanes replied:
Done and commited

cwinters8 replied:
@MyDanes this hasn't been updated in `about/index.html` yet

cwinters8 replied:
Oh, you changed this in the header that actually shows up on the webpage - you don't want that.

cwinters8 replied:
```suggestion
    <title>About | Doodle Dandy</title>
```

MyDanes replied:
Fixed and submitted

cwinters8 commented on breed/index.html:
```diff
@@ -0,0 +1,106 @@
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>About Doodle Dandy | Doodle Dandy</title>
```
Title needs to be updated to `Breed Info | Doodle Dandy`

MyDanes replied:
<title>Aussiedoodles | Doodle Dandy</title> thats what i have  confused where the above is

cwinters8 commented on current/index.html:
```diff
@@ -0,0 +1,106 @@
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>About Doodle Dandy | Doodle Dandy</title>
```
Title needs to be updated to `Current Litter | Doodle Dandy`

MyDanes replied:
Complete and thank you

cwinters8 reviewed (changes requested):
See comments

MyDanes committed:
Fixed Menus & Bernedoodle Info

6.1.2025 - Justice

Tested changes in local copy with the preview function

MyDanes committed:
Fixed casing and updated about

fixed the inconsistant Casing for AussieDoodles
Updated about us

6.1.2025 Justice

MyDanes committed:
Fixed about us title

6.1.2025 Justice

MyDanes committed:
updated Current Litter

6.1.2025 Justice

MyDanes committed:
Updated Titles on all pages & worked on reviews

6.1.2025 Justice

MyDanes committed:
Adding Reviews and divider

6.1.2025 Justice

MyDanes committed:
Updated Home page

Yes thats right I even figured out how to embed a video :)

6.1.2025 Justice

MyDanes committed:
LAST for tonight - more homepage tweaks

6.1.2025 Justice

MyDanes committed:
FINAL FINAL for the night - found a great embed

Got a photo album embed with pixieset to share photo albums of pups

6.1.2025 Justice

cwinters8 commented on assets/images/GoodDogDD.png:
Can you make the background of this image transparent instead of black? It would look a lot better

MyDanes replied:
Done

cwinters8 commented on about/index.html:
```diff
@@ -0,0 +1,60 @@
+      </nav>
+    </header>
+    <main>
+      <h1>About | Doodle Dandy</h1>
```
This should be more like this - the `|` character only belongs in the `title` tag, which shows up in the browser tab and when the page is added to bookmarks.

```suggestion
      <h1>About us</h1>
```

cwinters8 commented on bernedoodle/index.html:
```diff
@@ -0,0 +1,170 @@
+      />
+
+      <p>
+        Whether as a family pet or a therapy animal, bernedoodles inherit a
```
be consistent with how you capitalize the breed names
```suggestion
        Whether as a family pet or a therapy animal, Bernedoodles inherit a
```

MyDanes replied:
done

cwinters8 commented on contact/index.html:
```diff
@@ -31,7 +31,12 @@
       <nav>
         <a href="/">Home</a>
-        <a href="/contact" class="current">Contact us</a>
+        <a href="/bernedoodle">Bernedoodle Info</a>
+        <a href="/breed">AussieDoodle Info</a>
```
```suggestion
        <a href="/bernedoodle">Bernedoodles</a>
        <a href="/breed">Aussiedoodles</a>
```

MyDanes replied:
done

cwinters8 commented on contact/index.html:
```diff
@@ -31,7 +31,12 @@
+        <a href="/breed">AussieDoodle Info</a>
+        <a href="/about">About us</a>
+        <a href="/current" class="current">Current Litter</a>
+        <a href="/famstories">Reviews</a>
```
Why did you decide to call this directory/page `/famstories` instead of `/reviews`, when the page title is `Reviews`?

MyDanes replied:
that was a mistake and i dont know how to fix it

MyDanes replied:
updated

cwinters8 commented on current/index.html:
```diff
@@ -0,0 +1,71 @@
+      <img src="../assets/images/header.png" alt="Doodle Dandy" />
+      <nav>
+        <a href="/">Home</a>
+        <a href="/bernedoodle">Bernedoodle Info</a>
+        <a href="/breed">AussieDoodle Info</a>
```
```suggestion
        <a href="/bernedoodle">Bernedoodles</a>
        <a href="/breed">Aussiedoodles</a>
```

MyDanes replied:
fixed

cwinters8 commented on current/index.html:
```diff
@@ -0,0 +1,71 @@
+      </nav>
+    </header>
+    <main>
+      <h1>Current Litter | Doodle Dandy</h1>
```
Don't use the `|` character like this in the actual page content.
```suggestion
      <h1>Current Litter</h1>
```

MyDanes replied:
FIXED

MyDanes replied:
sorry cap lock was on I did not notice

cwinters8 commented on about/index.html:
```diff
@@ -0,0 +1,60 @@
+        <a href="/bernedoodle">Bernedoodle Info</a>
+        <a href="/breed">AussieDoodle Info</a>
+        <a href="/about">About us</a>
+        <a href="/current" class="current">Current Litter</a>
```
For the `nav` section on each page, you need to move the `class="current"` attribute to the link for the active page. So for this one, it should move to the `/about` link.

```suggestion
        <a href="/about" class="current">About us</a>
        <a href="/current">Current Litter</a>
```

MyDanes replied:
THATS how to fix the damn yellow box haha

cwinters8 commented on bernedoodle/index.html:
```diff
@@ -0,0 +1,170 @@
+    <header>
+      <img src="../assets/images/header.png" alt="Doodle Dandy" />
+      <nav>
+        <a href="/">Home</a>
+        <a href="/bernedoodle">Bernedoodle Info</a>
+        <a href="/breed">AussieDoodle Info</a>
+        <a href="/about">About us</a>
+        <a href="/current" class="current">Current Litter</a>
+        <a href="/famstories">Reviews</a>
+        <a href="/contact">Contact us</a>
```
I would take the word Info out of the breed links. Also fix the `current` link

```suggestion
        <a href="/">Home</a>
        <a href="/bernedoodle" class="current">Bernedoodles</a>
        <a href="/breed">Aussiedoodles</a>
        <a href="/about">About us</a>
        <a href="/current">Current Litter</a>
        <a href="/famstories">Reviews</a>
        <a href="/contact">Contact us</a>
```

cwinters8 commented on breed/index.html:
I would change the name of this directory to `aussiedoodle`. Make sure you update all the links from `/breed` to `/aussiedoodle` as well.

MyDanes replied:
complete good suggestion

cwinters8 commented on breed/index.html:
```diff
@@ -0,0 +1,108 @@
+        <a href="/about">About us</a>
+        <a href="/current" class="current">Current Litter</a>
+        <a href="/famstories">Reviews</a>
+        <a href="/contact">Contact us</a>
```
```suggestion
        <a href="/">Home</a>
        <a href="/bernedoodle">Bernedoodles</a>
        <a href="/breed" class="current">Aussiedoodles</a>
        <a href="/about">About us</a>
        <a href="/current">Current Litter</a>
        <a href="/famstories">Reviews</a>
        <a href="/contact">Contact us</a>
```

MyDanes replied:
done

cwinters8 commented on breed/index.html:
```diff
@@ -0,0 +1,108 @@
+      </nav>
+    </header>
+    <main>
+      <h1>Aussiedoodle Breed Info</h1>
```
In `bernedoodle/index.html` your header is `Bernedoodle Information`. Either use `Breed Info` or `Information` on both pages - just be consistent.

MyDanes replied:
fixed

cwinters8 commented on famstories/index.html:
```diff
@@ -0,0 +1,105 @@
+      <img src="../assets/images/header.png" alt="Doodle Dandy" />
+      <nav>
+        <a href="/">Home</a>
+        <a href="/bernedoodle">Bernedoodle Info</a>
+        <a href="/breed">AussieDoodle Info</a>
+        <a href="/about">About us</a>
+        <a href="/current" class="current">Current Litter</a>
+        <a href="famstories">Reviews</a>
```
```suggestion
        <a href="/bernedoodle">Bernedoodles</a>
        <a href="/breed">Aussiedoodles</a>
        <a href="/about">About us</a>
        <a href="/current">Current Litter</a>
        <a href="famstories" class="current">Reviews</a>
```

cwinters8 commented on index.html:
```diff
@@ -30,17 +30,31 @@
       <img src="./assets/images/header.png" alt="Doodle Dandy" />
       <nav>
-        <a href="/" class="current">Home</a>
+        <a href="/">Home</a>
+        <a href="/bernedoodle">Bernedoodle Info</a>
+        <a href="/breed">AussieDoodle Info</a>
+        <a href="/about">About us</a>
+        <a href="/current" class="current">Current Litter</a>
```
```suggestion
        <a href="/" class="current">Home</a>
        <a href="/bernedoodle">Bernedoodles</a>
        <a href="/breed">Aussiedoodles</a>
        <a href="/about">About us</a>
        <a href="/current">Current Litter</a>
```

cwinters8 commented on about/index.html:
```diff
@@ -0,0 +1,60 @@
+      <nav>
+        <a href="/">Home</a>
+        <a href="/bernedoodle">Bernedoodle Info</a>
+        <a href="/breed">AussieDoodle Info</a>
```
```suggestion
        <a href="/bernedoodle">Bernedoodles</a>
        <a href="/breed">Aussiedoodles</a>
```

MyDanes committed:
fixed title

about | Doodle Dandy - i fixed in wrong spot first time

cwinters8 commented on index.html:
```diff
@@ -50,10 +64,6 @@ <h2>Links</h2>
             >Available pets</a
```
I might move the `Available pets` link into your navigation or something - its a little lost at the bottom of this main page now there's a lot of extra content.

MyDanes replied:
Good idea done

cwinters8 commented on bernedoodle/index.html:
```diff
@@ -0,0 +1,170 @@
+    <main>
+      <h1>Bernedoodle Information</h1>
+
+      <h3>Welcome to the Wonderful World of Bernedoodles!</h3>
```
I would probably just make this a bolded element instead of a header - it looks awkwardly large on the page
```suggestion
      <strong>Welcome to the Wonderful World of Bernedoodles!</strong>
```

MyDanes replied:
thank you fixed

MyDanes committed:
Fix Class Current

6.2 Justice

MyDanes committed:
About Us

cwinters8 commented on famstories/index.html:
```diff
@@ -0,0 +1,105 @@
+
+
+
+   <h2>
+      <ul>
+        <li>
+          <i class="fa-solid fa-envelope fa-pad-right"></i>
+          <a href="mailto:adopt@doodledandy.dog">adopt@doodledandy.dog</a>
+        </li>
+        <li>
+          <i class="fa-solid fa-phone fa-pad-right"></i>
+          <a href="tel:+14174209882">417-420-9882</a>
+        </li>
+      </ul>
```
This is invalid html - at minimum, get rid of the unclosed `h2` element.
```suggestion

```

You already have a Contact page - get rid of this whole section.

MyDanes replied:
Done

cwinters8 commented on about/index.html:
```diff
@@ -0,0 +1,60 @@
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>About | Doodle Dandy | Doodle Dandy</title>
```
You don't need `| Doodle Dandy` twice.
```suggestion
    <title>About | Doodle Dandy</title>
```

cwinters8 replied:
This is still not fixed

cwinters8 commented on famstories/index.html:
```diff
@@ -34,8 +34,8 @@
-        <a href="/current" class="current">Current Litter</a>
-        <a href="famstories">Reviews</a>
+        <a href="/current">Current Litter</a>
+        <a href="famstories"class="current">Reviews</a>
```
There needs to be a space before the word `class` here
```suggestion
        <a href="famstories" class="current">Reviews</a>
```

cwinters8 commented on index.html:
```diff
@@ -30,11 +30,11 @@
       <img src="./assets/images/header.png" alt="Doodle Dandy" />
       <nav>
-        <a href="/">Home</a>
+        <a href="/"class="current">Home</a>
```
same here
```suggestion
        <a href="/" class="current">Home</a>
```

MyDanes committed:
Update GoodDogDD.png

MyDanes committed:
Update current/index.html

Co-authored-by: Clark Winters <github@clarkwinters.com>

MyDanes committed:
remove "Info"

MyDanes committed:
Merge branch 'DDandy' of https://github.com/DevonFarm/doodle-dandy into DDandy

MyDanes committed:
Fixed Info & Reviews

MyDanes committed:
Casing for Aussiedoodles & Bernedoodles fixed

MyDanes committed:
updaed breed to aussiedoodle directory & menu

MyDanes committed:
fixed available pups

MyDanes reviewed (commented):
Thank you Clark - I think I got it all :)

MyDanes committed:
Fixed Bernedoodle directory first letter lower

MyDanes committed:
s

MyDanes committed:
vfg

MyDanes committed:
Merge branch 'DDandy' of https://github.com/DevonFarm/doodle-dandy into DDandy

MyDanes committed:
Bernedoodle page changes

MyDanes committed:
Merge branch 'DDandy' of https://github.com/DevonFarm/doodle-dandy into DDandy

MyDanes committed:
updated dividers / photos and content

6.4.2025 Justice

MyDanes committed:
image change

MyDanes committed:
Content updates

cwinters8 commented on .github/workflows/web.yml:
```diff
@@ -7,7 +7,7 @@ permissions:
   pages: write
   id-token: write
-# Allow one concurrent deployment
+# Allow one conavailable deployment
```
I think you might have done a find and replace here :)

MyDanes replied:
i did :( fail

cwinters8 commented on CNAME:
```diff
@@ -0,0 +1 @@
+doodleloverz.com
```
Are you planning on changing domains?

MyDanes replied:
NO - that was me trying to have a spot to check it worked online - defo NOT - doodledandy.dog is going live

cwinters8 commented on index.html:
```diff
@@ -30,29 +30,38 @@
       <h1>Welcome!</h1>
+      
       <p>
-        Thanks for considering Doodle Dandy for your next family pet. Please
-        check out the links below to continue your journey
-        <i class="fa-solid fa-paw"></i>
+     <p>  Welcome to Doodle Dandy:</p>
```
You have two welcome messages here, and for some reason the new one ends in a colon. Get rid of either one or the other, and probably change the colon to an exclamation point `!` if you keep that one.

MyDanes replied:
redundancy removed

cwinters8 commented on contact/index.html:
```diff
@@ -31,11 +31,19 @@
     </header>
     <main>
       <h1>Contact us</h1>
+      <p>
+        <i class="fa-solid fa-paw"></i>
+      </p>
```
I wouldn't do these paw icons at the top of every page's content, especially since you're using the same icon in lists.

MyDanes replied:
was accident removed

cwinters8 commented on about/index.html:
```diff
@@ -0,0 +1,110 @@
+
+How are we different?
+
+<ul>
+  <li><i class="fa-solid fa-paw"></i>&nbsp;&nbsp;Gift Bag: We send home a gift bag that includes the popular Snuggle Puppy and many other goodies to help your pup transition to your home</li>
+  <li><i class="fa-solid fa-paw"></i>&nbsp;&nbsp;Communication: Once your adoption application is received, we will provide photos, video, and even Facetime with the puppy as we approach adoption day</li>
+<li><i class="fa-solid fa-paw"></i>&nbsp;&nbsp;Education: We have a ‘Congrats file’ that’s full of great information, including:</li>
+</ul>
+ <br/>
+</pre>o What’s the best way to travel with my puppy?<br/>
+</pre>o What do I need to know about the drive/flight home?<br/>
+</pre>o We share tips to make the transition to home as smooth as possible, while also creating good long-term habits<br/>
+</pre>o Grooming guidelines<br/>
+</pre>o Where do I start with toys?<br/>
+</pre>o Why crate train?<br/>
+ 
+<ul>
+  <li><i class="fa-solid fa-paw"></i>&nbsp;&nbsp;Home Raised: The puppies are always raised in my home, not in a separate kennel or building.</li>
+  <li><i class="fa-solid fa-paw"></i>&nbsp;&nbsp;Socialized: The puppies have many experiences to help them prepare for their new life with you, including other people / pets and activities</li>
+  <li><i class="fa-solid fa-paw"></i>&nbsp;&nbsp;Crate Training: Puppies are crate-trained with their Snuggle Puppy and toys before they come to your 
+    home to ensure a smooth transition. Here are a few reasons why this is such a huge head start:</li>
+
+</ul>
+<ul> 
+ 
+  <br/>
+</pre>o They will eat their food in their crates and have excellent eating habits<br/>
+</pre>o They will eat their food in their crates and have excellent eating habits<br/>
+</pre>o Potty training will be well underway<br/>
+<ul>
+  <li><i class="fa-solid fa-paw"></i>&nbsp;&nbsp;Health Tested: Their parents have been health tested with guaranteed pedigree</li>
+  <li><i class="fa-solid fa-paw"></i>&nbsp;&nbsp;Training: We will connect you with an excellent online trainer at a discounted rate. 
+    They have videos and information to walk you through every puppy habit or issue that you can think of.</li>
+    <li><i class="fa-solid fa-paw"></i>&nbsp;&nbsp;Transparency: We encourage our families to connect through our 
+      Facebook group. We want people to be able to ask questions and see littermates if they’d like to! We also confess the best part of our days is when we get to see "pupdates" from our families</li>
+
+</ul>
```
These lists look bad:

![image](https://github.com/user-attachments/assets/e46ddf0b-ca0d-48ca-99bd-4fb1a4d2a22c)

A lot of the HTML is invalid. I'm not sure what you're trying to do with `</pre>` etc. Make these into valid `<li>` elements like the other list items, and take out the `o ` you are prepending each item with.

MyDanes replied:
all fixed :) Thank you

MyDanes commented:
all addressed :) sending update

MyDanes committed:
all commented changes addresseed

Including
Paw print issue
Nightmare about page
css updated
fixed the mess of my <ul>
fixed cname

cwinters8 commented on about/index.html:
```diff
@@ -0,0 +1,104 @@
+ <li class="indent">o We share tips to make the transition to home as smooth as possible while also creating good long-term habits</li>
+ <li class="indent">o Grooming guidelines</li>
+ <li class="indent">o Where do I start with toys?</li>
+ <li class="indent">o Why crate train?</li>
```
Get rid of all these `o ` characters. If the `li` elements aren't showing up with a circle it means you need to fix the styling in CSS.

MyDanes replied:
oh no i typed those - trying to make them distinct as sub lists 
the ones farther indented

MyDanes replied:
gone

MyDanes committed:
Update about/index.html

Co-authored-by: Clark Winters <github@clarkwinters.com>

MyDanes committed:
removed the small circles from about

MyDanes committed:
Merge branch 'DDandy' of https://github.com/DevonFarm/doodle-dandy into DDandy

MyDanes committed:
fixed class needing space

MyDanes committed:
concurrent typo on web.yml

MyDanes committed:
Finished Reviews

cwinters8 commented on CNAME:
You don't need this file - doodledandy.dog is already configured. Please delete it

MyDanes replied:
COmplete

cwinters8 commented on about/index.html:
```diff
@@ -0,0 +1,104 @@
+  <li><i class="fa-solid fa-paw"></i>&nbsp;&nbsp;Gift Bag: We send home a gift bag that includes the popular Snuggle Puppy and many other goodies to help your pup transition to your home</li>
+  <li><i class="fa-solid fa-paw"></i>&nbsp;&nbsp;Communication: Once your adoption application is received, we will provide photos, video, and even Facetime with the puppy as we approach adoption day</li>
+<li><i class="fa-solid fa-paw"></i>&nbsp;&nbsp;Education: We have a ‘Congrats file’ that’s full of great information, including:</li>
+ <li class="indent"> What is the best way to travel with my puppy?</li>
+ <li class="indent"> What do I need to know about the drive/flight home?</li>
+ <li class="indent"> We share tips to make the transition to home as smooth as possible while also creating good long-term habits</li>
+ <li class="indent"> Grooming guidelines</li>
+ <li class="indent"> Where do I start with toys?</li>
+ <li class="indent"> Why crate train?</li>
```
You need this list to be in a new `ul` element with the `indent` class.

Like this:

```suggestion
  <ul class="indent">
    <li>What is the best way to travel with my puppy?</li>
    <li>What do I need to know about the drive/flight home?</li>
    <li>We share tips to make the transition to home as smooth as possible while also creating good long-term habits</li>
    <li>Grooming guidelines</li>
    <li>Where do I start with toys?</li>
    <li>Why crate train?</li>
  </ul>
```

cwinters8 replied:
Also the line about `We share tips to make the transition to home as smooth as possible while also creating good long-term habits` doesn't make sense in the list.

cwinters8 commented on about/index.html:
```diff
@@ -0,0 +1,104 @@
+  <li><i class="fa-solid fa-paw"></i>&nbsp;&nbsp;Crate Training: Puppies are crate-trained with their Snuggle Puppy and toys before they come to your 
+    home to ensure a smooth transition. Here are a few reasons why this is such a huge head start:</li>
+
+ <li class="indent"> They will eat their food in their crates and have excellent eating habits</li>
+ <li class="indent"> Potty training will be well underway</li>
```
Same as above - put these in a new `ul` list.

cwinters8 commented on assets/styles/main.css:
```diff
@@ -3,15 +3,26 @@
+  text-indent: 1rem; /* Adjust the value as needed */
+}
+
+.indlng {
+  text-indent: 2.2rem; /* Adjust the value as needed */
+}
```
What is this for?

cwinters8 commented on assets/styles/main.css:
```diff
@@ -32,7 +43,32 @@ header > nav {
   padding: 0;
 }
 
-header > nav a.current {
+header > nav a.available {
```
This was probably from your global find/replace. Change it back to current so the top nav link styling isn't broken.

cwinters8 commented on index.html:
```diff
@@ -30,29 +30,38 @@
       <img src="./assets/images/header.png" alt="Doodle Dandy" />
       <nav>
-        <a href="/" class="current">Home</a>
+        <a href="/" class="available">Home</a>
```
You need to fix all of the current links from `available` to `current`. Please be careful when doing a global find replace like this.

cwinters8 commented on assets/styles/main.css:
```diff
@@ -3,15 +3,26 @@
   margin: 0.5rem 0;
   font-size: 1.3rem;
 }
+.indent {
+  text-indent: 1rem; /* Adjust the value as needed */
+}
```
This is incorrect. You don't need `text-indent`, you need `margin-left`.

The list items don't have a list icon because of the `list-style: none;` CSS rule applied to all the `ul` elements. To have one enabled for the indented items, add the `list-style: initial` rule.

```suggestion
.indent {
  margin-left: 1rem;
  list-style: initial;
}
```

MyDanes committed:
Update about/index.html

Co-authored-by: Clark Winters <github@clarkwinters.com>

MyDanes committed:
Update assets/styles/main.css

Co-authored-by: Clark Winters <github@clarkwinters.com>

MyDanes committed:
all new notes updated for menus and about us

MyDanes committed:
fixed the botched line

MyDanes committed:
Merge branch 'DDandy' of https://github.com/DevonFarm/doodle-dandy into DDandy

MyDanes committed:
css

cwinters8 commented on about/index.html:
```diff
@@ -57,7 +57,7 @@ <h1>About us</h1>
  <li class="indent"> What is the best way to travel with my puppy?</li>
  <li class="indent"> What do I need to know about the drive/flight home?</li>
- <li class="indent"> We share tips to make the transition to home as smooth as possible while also creating good long-term habits</li>
+ <li class="indent"> Go home guidance for the transition to your home </li>
```
This still sucks. Also if the format of the list is going to be questions, make them all questions consistently.

Get rid of the single empty space before the text in each of these elements. I also think somehow you overwrote the fix for the list somehow - the two indented lists on this page are still incorrect.

MyDanes committed:
sending my change again for about

MyDanes committed:
fixed UL

cwinters8 commented:
bugbot run

MyDanes committed:
Fixed Class issue on about us

MyDanes committed:
Complements of Gemini to fix the two items to questions :)

MyDanes committed:
missed one now fixed for about us questions

MyDanes committed:
missed one of the current class updates on index

MyDanes committed:
updated contact page

MyDanes committed:
Update Contact page

MyDanes committed:
Update index.html

MyDanes committed:
fixed contact card

MyDanes committed:
minor word change on contact

MyDanes committed:
removed waiting list from tag

MyDanes committed:
one addition

cwinters8 commented:
bugbot run

cwinters8 committed:
Fix indent styling

cwinters8 committed:
fix left margin indent

cwinters8 committed:
fix priorities paragraph element

cwinters8 commented on bernedoodle/index.html:
```diff
@@ -0,0 +1,181 @@
+        class="dog"
+      />
+      <p>
+        Depending on the size of the Poodle used (standard, miniature, or toy),
```
```suggestion
        Depending on the size of the parents,
```

cwinters8 committed:
Update bernedoodle/index.html
